test/helpers/repo.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { execSync } = require('child_process');
+const { CI } = process.env;
+
+/**
+ * exec sync and redirect output to stdio
+ */
+function exec(cmd) {
+  try {
+    console.log('RUN', cmd);
+    return execSync(cmd, { stdio: [0, 1, 2] });
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  }
+}
+
+function getCommitSHA() {
+  // fetching the commit SHA
+  const lastCommits = execSync('git log -n 2 --pretty=oneline')
+    .toString()
+    .split('\n')
+    .map(log => log.split(' ')[0]);
+
+  // the repo is in the detached state for CI
+  // we need to take a one commit before to take a commit that has been associated to the PR
+  return CI ? lastCommits[1] : lastCommits[0];
+}
```

Comment:
FYI recently learned about `git rev-parse` during my bundle work.  Can do nifty stuff like `git rev-parse HEAD` or `git rev-parse HEAD^` to get the commit SHA of HEAD, or the commit behind HEAD.

Reply:
Nice, will use that